stable/redis-cluster/templates/redis-sentinel-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: sentinel
+    role: service
+  name: redis-sentinel
+spec:
+  ports:
+    - port: 26379
+      targetPort: 26379
+  selector:
+    redis-sentinel: "true"
```

Comment:
It might be worth using the helm predefined values for selector to ensure that you select a specific release. In this case you cannot install the same chart in a given namespace.